Readability.js

```diff
@@ -329,10 +329,25 @@ Readability.prototype = {
   },
 
   _setNodeTag: function (node, tag) {
-    // FIXME this doesn't work on anything but JSDOMParser (ie the node's tag
-    // won't actually be set).
-    node.localName = tag.toLowerCase();
-    node.tagName = tag.toUpperCase();
+    this.log("_setNodeTag", node, tag);
+    if (node.__JSDOMParser__) {
```

Comment:
Which part does it not support? I think it should work in theory, but I'd be very surprised if it wasn't slower. That said, the benchmarks are super noisy on my machine. I get results anywhere between 170 and 250 ops/s for the JSDOMParser part on the reference benchmark, with an average of about 200, all on the same changeset.